src/file/upload.model.ts

```diff
@@ -15,6 +15,10 @@ export const uploadSchema: mongoose.Schema = new mongoose.Schema(
     },
     bucket: {
       type: String,
+    },
```

Comment:
`required` is defined on `uploadID` and on `name` but not on `key` and `bucket`.

Reply:
Fix indentation